modules/opening-default-network-ports-udn.adoc

```diff
@@ -0,0 +1,27 @@
+//module included in the following assembly:
+//
+// * networking/multiple_networks/primary_networks/about-user-defined-networks.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="opening-default-network-ports-udn_{context}"]
+= Opening default network ports on user-defined network pods
+
+By default, user-defined network pods cannot connect to some services, such as monitoring (including Prometheus and Alertmanager) or the {product-title} image registry. This is because UDN pods do not have access to the default network unless specific ports are explicitly opened.
```

Comment:
I think it is a bit vise-versa: user-defined pods are isolated from the default network, so to allow default network pods (like monitoring) to connect TO udn pods, you can use this annotation